feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go

```diff
@@ -170,7 +169,7 @@ func TestLinecardReboot(t *testing.T) {
 		}
 	}
 	if removableLinecard == "" {
-		t.Fatalf("Component(lc).Removable().Get(t): got none, want non-empty")
+		t.Skipf("No removable line card found for the testing")
```

Comment:
Having 0 removable linecards for a modular devce is an error.

Can you check if `arg.NumLinecards > 0` and then throw error if `removableLinecard == ""`

Reply:
Thanks for your comments. Updated script as below:
```go
		if *args.NumLinecards > 0 {
			t.Fatalf("No removable line card found for the testing on a modular device")
		} else {
			t.Skipf("No removable line card found for the testing")
		}
```